documentation/tenant/users.adoc

```diff
@@ -0,0 +1,65 @@
+= Users
+
+{ProductName} uses Red Hat Single Sign-On to manage users and credentials. You can create and delete {ProductName} users as well as change an existing password.
+
+[[_create_user]]
+== Creating a User
+
+Using the Red Hat Single Sign-On Administration Console you can create a new {ProductName} user and specify a password.
```

Comment:
"Red Hat Single Sign-On" Administration Console - This should be converted into a variable {AuthProductName} or something, since this should say KeyCloak here, but for versions built using the Red Hat product derived from KeyCloak, that name should be substituted with Red Hat Single Sign-On